changelogs/fragments/569-pkgng-add-upgrade-action.yaml

```
@@ -0,0 +1,6 @@
+---
+minor_changes:
+  - pkgng: Added support for upgrading all packages using ``name: *, state: latest``, similar to other package providers (https://github.com/ansible-collections/community.general/pull/569).
```

```suggestion
  - pkgng - added support for upgrading all packages using ``name: *, state: latest``, similar to other package providers (https://github.com/ansible-collections/community.general/pull/569).
```
Same for the others.

---

Updated, thanks!